mlrun/datastore/targets.py

```diff
@@ -618,3 +618,7 @@ def _get_target_path(driver, resource):
     # todo: handle ver tag changes, may need to copy files?
     name = f"{name}-{version or 'latest'}"
     return f"{data_prefix}/{kind_prefix}/{name}{suffix}"
+
+
+def resolve_framesd():
+    return os.environ.get("V3IO_FRAMESD") or config.v3io_framesd
```

Comment:
The value in `config.v3io_framesd` is already taken from `V3IO_FRAMESD`, why it this needed ?

Reply:
didnt work for me, maybe because i was setting the env after the mlrun import. anyway we do want to have some resolve in the future for all remote cluster URLs based on the dbpath (if its a remote one) so users dont need to manually set all the external API URLs